gdal/ogr/ogrsf_frmts/EFAL/ogrgeopackageutility.h

```diff
@@ -0,0 +1,47 @@
+/*****************************************************************************
+* Copyright 2016 Pitney Bowes Inc.
```

Comment:
Is this file needed? It appears to be the existing [ogrgeopackageutility.cpp](https://github.com/OSGeo/gdal/blob/master/gdal/ogr/ogrsf_frmts/gpkg/ogrgeopackageutility.cpp)  file from the gpkg driver with Pitney Bowes (c) on it instead of the original author @pramsey. 

If this is cruft it will have to be cleaned up. If it is used, it is not legitimate to take an existing file and change the copyright.

Reply:
This file is used in our driver. I have reverted back the copyright to original.

Reply:
Interesting... Anyway, code duplication inside the code base is not the right solution. It is quite strange that the linker didn't complain about symbol duplication. I'd say that those 2 files should be moved from ogr/ogrsf_frmts/gpkg to the top ogr/ directory